Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -284,6 +456,17 @@ YUI.add('ez-subitemlistview', function (Y) {
              * @type Array of {Object} array containing location structs
              */
             subitems: {},
+
+            /**
+             * Boolean to check if a subitem priority is currently selected.
+             *
+             * @attribute isOnePrioritySelected
+             * @default false
+             * @type Boolean
+             */
+            isOnePrioritySelected: {
```

Comment:
ok now I see the point of this attribute. I would just call it `editingPriority`